view/frontend/web/js/view/payment/method-renderer/adyen-paypal-method.js

```diff
@@ -12,11 +12,15 @@ define(
         'Magento_Checkout/js/model/quote',
         'Adyen_Payment/js/view/payment/method-renderer/adyen-pm-method',
         'Magento_Checkout/js/model/full-screen-loader',
+        'Adyen_Payment/js/model/adyen-payment-service',
+        'Magento_Checkout/js/model/error-processor',
     ],
     function(
         quote,
         adyenPaymentMethod,
-        fullScreenLoader
+        fullScreenLoader,
+        adyenPaymentService,
+        errorProcessor
     ) {
         return adyenPaymentMethod.extend({
             placeOrderButtonVisible: false,
```

Comment:
To be on the safe side, let's define `token` as a property with default value `null`.

```suggestion
            placeOrderButtonVisible: false,
            token: null,
```